aws/cloudformation-templates/base/ssm.yaml

```diff
@@ -16,9 +16,16 @@ Parameters:
     Description: Amplitude API key for product analytics and A/B testing results (optional).
     NoEcho: true
 
+  OptimizelySdkKey:
+    Type: String
+    Description: Optimizely SDK key for experimentation (optional).
+    NoEcho: true
+
 Conditions:
   HasAmplitudeApiKey:
     !Not [!Equals [!Ref AmplitudeApiKey, ""]]
+  OptimizelySdkKey:
```

Comment:
The condition name should be something like `HasOptimizelySdkKey` so it's different than the parameter name.